docs/contributing/ways_to_contribute.rst

```diff
@@ -12,44 +12,47 @@ Talk to us
 
 * Get support in our `Community Forums <http://community.learningequality.org/>`__.
 * Email us at info@learningequality.org
-* Sign up to receive developer announcements in `Google groups <https://groups.google.com/a/learningequality.org/forum/#!forum/dev>`__
 * Visit the ``#kolibri`` room on Freenode IRC
 
 Translate
 ---------
 
 Help us `translate the application on Crowdin <https://crowdin.com/project/kolibri>`__.
 
-Give Feedback
+Give feedback
 -------------
 
-`Report issues on github <https://github.com/learningequality/kolibri/issues>`__. Please search the existing issues first to see if it's already been reported.
+You can ask questions, make suggestions, and report issues in the `community forums <https://community.learningequality.org/>`__.
 
-If you are reporting a bug, please include:
+If you have found a bug and are comfortable using Github and Markdown, you can create a `Github issue <https://github.com/learningequality/kolibri/issues>`__. Please search the existing issues first to see if it's already been reported, and make sure to use the template.
 
-* Your operating system name and version.
-* Any details about your local setup that might be helpful in troubleshooting.
-* Detailed steps to reproduce the bug.
+If you are reporting an issue, please include:
+
+* Your operating system name and version
+* Any details about your local setup that might be helpful in troubleshooting
+* Detailed steps to reproduce the bug
 
```

Comment:
Or simpler:

> If you are reporting an issue, please make sure to fill in all the relevant parts in the **New issue** [template](https://github.com/learningequality/kolibri/issues/new)...? 🙂

Reply:
I do mention that above so you're right, this is redundant

Reply:
updated